src/main/resources/edu/cornell/kfs/fp/cu-ojb-fp.xml

```diff
@@ -1327,4 +1327,15 @@
 		<field-descriptor name="subAccountNumber" column="SUB_ACCT_NBR" jdbc-type="VARCHAR" />
 	</class-descriptor>
 
+    <class-descriptor class="edu.cornell.kfs.fp.businessobject.CreateAccountingDocumentFileEntry" table="CU_FP_CRTE_ACCT_DOC_FIL_T">
+        <field-descriptor name="fileName" column="FIL_NM" jdbc-type="VARCHAR" primarykey="true" index="true" />
```

Comment:
sorry I didn't mention it before, but it may be nice to include sequence ID field as the primary key, and make FIL_NM a unique constraint.

Reply:
I'll add the suggested columns and constraints.